aws-lc-sys/builder/main.rs

```diff
@@ -303,6 +362,8 @@ fn main() {
     println!("cargo:rerun-if-changed=builder/");
     println!("cargo:rerun-if-changed=aws-lc/");
     println!("cargo:rerun-if-env-changed=AWS_LC_SYS_STATIC");
+    println!("cargo:rerun-if-env-changed=AWS_LC_SYS_INTERNAL_NO_PREFIX");
+    println!("cargo:rerun-if-env-changed=AWS_LC_RUST_INTERNAL_BINDGEN");
```

Comment:
We should really wrap the env var functions, similar [to s2n-tls](https://github.com/aws/s2n-tls/blob/d9169425b7db7aa28c52a6efb8c087c68d1dbd16/bindings/rust/s2n-tls-sys/build.rs#L15-L24), just so we don't accidentally miss one of the variables we're consuming.

Reply:
Addressed in latest commit.